### PR TITLE
Add test cases set name

### PR DESCRIPTION
**Pull request by RyanDJLee**
- Add test case for set-name visitor for single unassigned variable.
- Add test case for set-name visitor using hypothesis for random number of assigned variables.

### CODE REVIEW

File: tests/type_inference/test_literals.py [hunk 1]
```diff
@@ -125,6 +125,31 @@ def test_set_env():
...
+
+
+@given(cs.random_dict_variable_value(min_size=1))
+def test_set_name_assigned(variables_dict):
+    """Test visitor for name nodes representing a variables with assigned values in module."""
+    program = ""
+    # parse dictionary into input program
+    for variable_name in variables_dict:
+        assume(not iskeyword(variable_name))
+        program += variable_name + " = " + repr(variables_dict[variable_name]) + "\n" + variable_name + "\n"
```
Comment (david-yz-liu) [inline tests/type_inference/test_literals.py]: There's a helper for this now, too?

Comment (RyanDJLee) [inline tests/type_inference/test_literals.py]: There is not for this one, the difference is that the helper has a random number of single-line-single-assignment statements, whereas in this test case I want the program to have the assignments as well a single line afterwards with just the variable name in order to get the actual Name nodes in the AST once parsed.

Comment (david-yz-liu) [inline tests/type_inference/test_literals.py]: In this case, you should use your helper first, and then evaluate single expressions with each name afterwards.

File: tests/type_inference/test_literals.py [hunk 2]
```diff
@@ -125,6 +125,31 @@ def test_set_env():
     for value in global_values: assert isinstance(value, TypeVar)
 
 
+def test_set_name_unassigned():
+    """Test visitor for name nodes representing a single unassigned variable in module."""
+    program = "george"
+    module = _parse_text(program)
+    name_nodes = [n for n in module.nodes_of_class(astroid.Name)]
```
Comment (david-yz-liu) [inline tests/type_inference/test_literals.py]: See

File: tests/type_inference/test_literals.py [hunk 3]
```diff
@@ -125,6 +125,31 @@ def test_set_env():
     for value in global_values: assert isinstance(value, TypeVar)
 
 
+def test_set_name_unassigned():
```
Comment (david-yz-liu) [inline tests/type_inference/test_literals.py]: This test should be done using hypothesis.

File: tests/type_inference/test_literals.py (hunk 2 above)
Comment (david-yz-liu) [inline tests/type_inference/test_literals.py]: Also see my comments in the other pull request.

File: tests/type_inference/test_literals.py [hunk 4]
```diff
@@ -124,6 +124,28 @@ def test_set_env():
     for value in local_values: assert isinstance(value, TypeVar)
     for value in global_values: assert isinstance(value, TypeVar)
 
+@given(hs.text(alphabet="abcdefghijklmnopqrstuvwxyz", min_size=1))
+def test_set_name_unassigned(variable_name):
+    """Test visitor for name nodes representing a single unassigned variable in module."""
+    program = variable_name
+    module = _parse_text(program)
+    name_nodes = [n for n in module.nodes_of_class(astroid.Name)]
```
Comment (david-yz-liu) [inline tests/type_inference/test_literals.py]: @RyanDJLee you have not yet fixed this.

**Pull request merged**